### DISCUSSION TITLE
Is this package available for mac os ??

### DISCUSSION THREAD

**Discussion by PrazolMalla** [TomSchimansky/CustomTkinter#2071] [Q&A]
Is this package available for mac os ??

**Comment by Akascape** [comment]
@PrazolMalla yes

**Reply by PrazolMalla** [reply]
@Akascape can you help me with this warning 
DEPRECATION WARNING: The system version of Tk is deprecated and may be removed in a future release. Please don't rely on it. Set TK_SILENCE_DEPRECATION=1 to suppress this warning. 
 
I have set the given code in bash but didn't work

**Reply by Akascape** [reply]
@PrazolMalla You have to update tkinter.

**Reply by PrazolMalla** [reply]
@Akascape thank you so much

**Reply by Lautarob1** [reply]
I'm starting using it with macOS. Hopefully, it will not produce the segmentation errors that are fairly common when using ttk.
